v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/convertors/ChangeEventSpannerUtils.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.spanner.migrations.convertors;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Value;
+import com.google.cloud.teleport.v2.spanner.ddl.Table;
+import com.google.cloud.teleport.v2.spanner.migrations.exceptions.ChangeEventConvertorException;
+import com.google.cloud.teleport.v2.spanner.type.Type;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Class for util methods processing change events and Spanner structures (ex - {@link
+ * com.google.cloud.spanner.Mutation}, {@link com.google.cloud.spanner.Key}).
+ */
+public class ChangeEventSpannerUtils {
+  public static void populateMutationBuilderWithEvent(
```

Comment:
This method has a very awkward signature for it to be its own public method in a separate class (it was still okay when it was a private method in the earlier class).

It's generally a good practice to think about the interface of public methods while refactoring! :)

In this case, the method is taking a `builder` as a parameter and updating it using the `changeEvent` provided, which is another parameter. 

I have two suggestions here - 

1. At the minimum, this method should **return** the `Mutation.WriteBuilder` instead of taking it as a parameter.
2. Much more preferably, this method should build the mutation and return it? Logically speaking, it makes sense to have a class which takes in `JsonNode` object and returns the `Mutation` formed from it?

Reply:
Agreed, refactored to point 2